### PR TITLE
Added Documentation on How-To run DefectDojo Hook Locally

### PR DESCRIPTION
**Pull request by JohannesZahn**
## Description

Provided detailed description on how to make the DefectDojo Hook run locally for development.

### Checklist

* [x] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [x] Make sure `npm test` runs for the whole project.
* [x] Make codeclimate checks happy

### CODE REVIEW

File: hooks/persistence-defectdojo/README.md [hunk 1]
```diff
@@ -136,3 +136,23 @@ helm upgrade --install dd secureCodeBox/persistence-defectdojo \
...
+included [Minio Instance](https://docs.securecodebox.io/docs/getting-started/installation/#accessing-the-included-minio-instance)
+and upload them to a GitHub gist.
+
+4. Set the following environment variables
+
+- DEFECTDOJO_URL (e.g http://192.168.0.228:8080);
+- DEFECTDOJO_USERNAME (e.g admin)
+- DEFECTDOJO_APIKEY= (e.g. b09c.., can be fetched from the DefectDojo Settings)
+- IS_DEV=true
+- SCAN_NAME (e.g nmap-scanme.nmap.org)
```
Comment (rseedorff) [inline hooks/persistence-defectdojo/README.md:155]: What is the `SCAN_NAME` here for? Do i need to add the corresponding Name of the scan which user starts in step 2 or could it be any random scan name? Maybe a comment would help to clearify that :)

File: hooks/persistence-defectdojo/README.md [hunk 2]
```diff
@@ -136,3 +136,23 @@ helm upgrade --install dd secureCodeBox/persistence-defectdojo \
...
+
+5. After running gradle build, start the DefectDojoPersistenceProvider with the following CLI Arguments:
```
Comment (rseedorff) [inline hooks/persistence-defectdojo/README.md:157]:
```
gradle ?target?
```

File: hooks/persistence-defectdojo/README.md [hunk 3]
```diff
@@ -136,3 +136,23 @@ helm upgrade --install dd secureCodeBox/persistence-defectdojo \
...
+{Raw Result Download URL} {Findings Download URL} {Raw Result Upload URL} {Findings Upload URL}. E.g. https://gist.githubusercontent.com/.../scanme-nmap-org.xml https://gist.githubusercontent.com/.../nmap-findings.json https://httpbin.org/put https://httpbin.org/put
```
Comment (rseedorff) [inline hooks/persistence-defectdojo/README.md:158]:
```
    ```bash
    jar -java ??? {Raw Result Download URL} {Findings Download URL} {Raw Result Upload URL} {Findings Upload URL}
    ???
    ```
```

**Review by rseedorff — CHANGES_REQUESTED** [review]
Great one! 👍🏻  
To finalize it some small additional hints and code snippets would be great ;-)

File: hooks/persistence-defectdojo/README.md (hunk 1 above)
Comment (JohannesZahn) [reply hooks/persistence-defectdojo/README.md]: Done

File: hooks/persistence-defectdojo/README.md (hunk 2 above)
Comment (JohannesZahn) [reply hooks/persistence-defectdojo/README.md]: Done

File: hooks/persistence-defectdojo/README.md (hunk 3 above)
Comment (JohannesZahn) [reply hooks/persistence-defectdojo/README.md]: Done

**Review by rseedorff — APPROVED** [review]

**Pull request merged**